src/index.ts

```diff
@@ -78,19 +78,20 @@ const stripChainspecJsonName = (chain: string) => {
  * @param chain
  */
 const getChainspec = (image: string, chain: string) => {
-  let res;
+  const outputChainSpec = `${chain}-${new Date().toISOString().slice(0, 10)}.json`;
```

Comment:
best to put temp files in temp dir

https://github.com/open-web3-stack/parachain-launch/blob/144b34a9940cae2bf2f2531ceb1ad23d621fb914/src/index.ts#L223

Reply:
Sure - adjusted